README.md

```diff
@@ -69,6 +69,23 @@ installation and configuration.
 [packaging]: https://github.com/Azure/iotedge/blob/main/doc/packaging.md
 [updating-guide]: https://learn.microsoft.com/azure/iot-edge/how-to-update-iot-edge?view=iotedge-1.4&tabs=ubuntu#special-case-update-from-10-or-11-to-latest-release
 
+
+Branching Strategy and Timelines
+===============================
+
+| Yoto | IoT Edge version | Branch Name | Branch Status |
+| :- | :- | :- | :- |
+| Dunfell | 1.4.x  | main | Active and maintained |
+| Dunfell | 1.4.x  | dunfell | Active and maintained |
+| Dunfell | 1.1.x  | dunfell-1.1 | 1.1.x is deprecated in Dec'22. Not maintained |
+| Kirkstone | 1.1.x | N/A | 1.1.x is deprecated in Dec'22 |
+| Kirkstone | 1.4.x | N/A | ETA TBD |
```

Comment:
I think we can drop the Kirkstone + 1.1 entry and just move the Kirkstone + 1.4 entry to be the first one.  I'm assuming that if (when) we add it the codeflow would be to first update the main (master) branch and then fork a Kirkstone branch.